src/pages/forgotPassword/styles.scss

```diff
@@ -0,0 +1,91 @@
+.column {
+  float: left;
+  width: 50%;
+}
+
+.row:after {
+  display: table;
+  clear: both;
+}
+.left-column {
+  background-color: var(--color-white);
+  height: 100vh;
+}
+.right-column {
+  background-color: var(--color-light-blue);
+  height: 100vh;
+}
+.circles {
+  width: 133px;
+  height: 78px;
+  margin: auto;
+  display: block;
+  background-image: url('../../assets/smilies.png');
+  background-repeat: no-repeat;
+  border: 0px;
+}
+
+.i6 {
+  background-image: url('../../assets/i6.png');
+  background-repeat: no-repeat;
+  display: block;
+  margin: 0 3.4rem auto;
+  width: 500px;
+  height: 625px;
+  border: 0px;
+}
+.explanation {
+  font-size: 14px;
+  line-height: 26px;
+}
+
+.forgot {
+  font-size: 11px;
+  line-height: 13px;
+  text-align: center;
+}
+
+#apple-store {
```

Comment:
I believe you could remove this styling if you just use `<img>` tags for every image instead of using `background-image`, just like:
` <img src={playStoreIcon} alt="Some alt text" />`